### ISSUE TITLE
Uncaught TypeError: App.getFrameTime is not a function

### ISSUE THREAD

**Issue by xhlwill** [xjsframework/xjs#127]
I wrote source plugin following tutorial, get resolution, framesrendered, but not the frame time, what's the right way?
```
(function() {
  'use strict';
  var xjs = require('xjs');
  var App = new xjs.App();

  xjs.ready().then(function() {

    App.getResolution().then(function(res) {
      var height = res.getHeight();
      var width = res.getWidth();
      var p = document.createElement('p');
      p.textContent = 'Height:' + height + ',Width:' + width;
      document.body.appendChild(p);
    });

    App.getFramesRendered().then(function(res) {
      var p = document.createElement('p');
      p.textContent = 'framesrendered:' + res;
      document.body.appendChild(p);
    });
    
    App.getFrameTime().then(function(res) {
      var p = document.createElement('p');
      p.textContent = 'frametime:' + res;
      document.body.appendChild(p);
    });
  });
})();
```
![xjs-frame](https://cloud.githubusercontent.com/assets/14025269/21542859/183013ce-cdfc-11e6-9322-a201075bbc00.png)

**Comment by virn** [comment]
@xhlwill I checked the framework and noticed that there's a small difference between the reference on the tutorial and the actual code. For now, please use `getFrametime` instead.

We'll have to update this too to avoid this confusion.

**Comment by xhlwill** [comment]
@virn works now, thanks for the information

**Comment by virn** [comment]
Fixed on 3a2e81c
